beacon-chain/db/iface/interface.go

```diff
@@ -129,6 +129,7 @@ type NoHeadAccessDatabase interface {
 	// Custody operations.
 	UpdateSubscribedToAllDataSubnets(ctx context.Context, subscribed bool) (bool, error)
 	UpdateCustodyInfo(ctx context.Context, earliestAvailableSlot primitives.Slot, custodyGroupCount uint64) (primitives.Slot, uint64, error)
+	UpdateEarliestAvailableSlot(ctx context.Context, earliestAvailableSlot primitives.Slot) (primitives.Slot, uint64, error)
```

Comment:
The outputs of `UpdateEarliestAvailableSlot` are never used. I think we can remove them for now.